backend/src/models/model_helpers.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+func StringPtr(s string) *string {
```

Comment:
We have the `UpdateStruct` function in `backend/src/models/resource.go`

Could you move that into this file? that func is also used in many other places.

Reply:
Sounds good, just pushed that change